docs/transpile/transpiler-plugins.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p><code>high_level_synthesis_plugin_names</code> requires the knowledge of types of <code>Operation</code>s that can be synthesized. I think for completeness would also include </p><p> </p><pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; from qiskit.transpiler.passes.synthesis.plugin import HighLevelSynthesisPluginManager
&gt;&gt;&gt; HighLevelSynthesisPluginManager().plugins.names()
</pre><p> </p><p>to list the names that are reuired in the entry point.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done.